src/test/scala/sigmastate/serialization/TypeSerializerSpecification.scala

```diff
@@ -93,12 +91,6 @@ class TypeSerializerSpecification extends SerializationSpecification {
       Array[Byte](TupleTypeCode, 5, SLong.typeCode, SLong.typeCode, SByte.typeCode, SBoolean.typeCode, SInt.typeCode))
   }
 
-  property("tuple of tuples crazy deep") {
```

Comment:
@greenhat I don't see it is replaced by something similar?

Reply:
I replaced it with two tests in `DeserializationResilience` (see last two tests https://github.com/ScorexFoundation/sigmastate-interpreter/blob/4eee4bf45c60d7a73bf6111ebb4ae22ddeb71a59/src/test/scala/sigmastate/serialization/DeserializationResilience.scala#L185 ). It's not exactly the same test, but I don't like this one, since it's kind of `reverse-engineered` from serialization format and thus is too fragile.